components/policy-mgt/org.wso2.carbon.policy.mgt.core/src/main/java/org/wso2/carbon/policy/mgt/core/dao/impl/ProfileDAOImpl.java

```diff
@@ -155,6 +155,29 @@ public boolean deleteProfile(Profile profile) throws ProfileManagerDAOException
     }
 
     @Override
+    public boolean deleteProfile(int profileId) throws ProfileManagerDAOException {
```

Comment:
deleteProfile does not sound like returning a boolean. BTW, do we really need to return true here if transaction is successfull?